.github/workflows/verify-bluehawk.yml

```
@@ -0,0 +1,21 @@
+name: Verify Bluehawk Was Run
+
+on: pull_request
+
+jobs:
+  check-bluehawk:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Ensure Bluehawk Script Was Run
+        working-directory: sync-todo/v2
+        run: |
+          dir1=client/
+          dir2=generated/
+          if [ `diff $dir1 $dir2 | grep -c 'Only'` == 0 ]
+          then 
+            echo "Passed"
+          else 
+            echo "Fail -- one or more files exist in /client that aren't in /generated."
```

Could we provide helpful info about how to fix this failure? Maybe something like:

```suggestion
            echo "Oops - looks like you updated files in the /client directory but they don't have matching updated files in the /generated directory. Please run the `bluehawk.sh` script in `sync-todo/v2` to generate updated output files."
```

(As a bonus, it might be nice to mention _which_ files got updated in `client` but aren't present in `generated`, but if that's annoying to do, it's not essential. And I'm not sure what this suggestion looks like in output - I'm not sure if we should split the echo across multiple lines?)